src/etc/inc/interfaces.inc

```diff
@@ -3111,6 +3111,13 @@ function interface_dhcpv6_configure($interface = 'wan', $wancfg)
     $wanif = get_real_interface($interface, "inet6");
     $dhcp6cconf = "";
     $dhcp6cconf .= "interface {$wanif} {\n";
+    
+    if (!empty($config['system']['global-v6duid'])) {
```

Comment:
Not sure about “global-“ suffix for a global value. How about ipv6duid ?

Reply:
I guess because when it was discusses elsewhere there was a lot of discussion of to where to put the DUID setting, some were unaware that you can only run one instance of dhcp6c and therefore the DUID is in fact global. It is whatever you wish to call it. :)